README.md

```diff
@@ -1,2 +1,32 @@
 # circleci-tools
-A collection of CircleCI tools to optimise build times and workflows 
+A collection of CircleCI tools to optimise build times and workflows
+
+## Requirements
+- bash
+- jq
+- wget
+- curl
+
+## Usage
+
+1. Add CIRCLE_TOKEN environment variable to your project
+2. Add `wget` and script execution when you wish to check for newer builds
+
+### Example CircleCI config
+
+```
+jobs:
+  test:
+    docker:
+      - image: jesiio/docker-git-awscli:latest
+    steps:
+      - run: 'wget -O - https://raw.githubusercontent.com/jesims/circleci-tools/master/cancel-redundant-builds.sh | bash'
```

Comment:
Could use `curl` here, may be simpler.
Wget is really better when you just want to easily download the file, and curl would be on more CircleCI images